ostree-raw-image.sh

```diff
@@ -652,7 +652,11 @@ ansible_become_pass=${EDGE_USER_PASSWORD}
 EOF
 
 # Test IoT/Edge OS
-sudo ANSIBLE_STDOUT_CALLBACK=yaml ansible-playbook -v -i "${TEMPDIR}"/inventory -e os_name=redhat -e ostree_commit="${UPGRADE_HASH}" -e ostree_ref="${REF_PREFIX}:${OSTREE_REF}" check-ostree.yaml || RESULTS=0
+if [[ "${ID}-${VERSION_ID}" == "rhel-8.5" ]]; then
```

Comment:
Did you REALLY tested locally? Your change will make test failed on RHEL 8.5. PLEASE run test locally before your send PR. Thanks.

Reply:
No, I didn't touch this file. This is the code change that I rebased from main.